doc/rustdocfx/workspace.go

```diff
@@ -494,15 +540,22 @@ type borrowedRef struct {
 	Type      typeEnum
 }
 
-func (t *borrowedRef) toString() string {
+func (t *borrowedRef) toString() (string, error) {
 	if t == nil {
-		return ""
+		return "", fmt.Errorf("error borrowRef.toString: unexpected nil")
+	}
+	typeString, err := t.Type.toString()
+	if err != nil {
+		return "", fmt.Errorf("error borrowRef.toString: %w", err)
+	}
+	lifetime := ""
+	if t.Lifetime != nil {
+		lifetime = *t.Lifetime + " "
 	}
-	// TODO: Error check for lifetime.
 	if t.IsMutable {
-		return fmt.Sprintf("&mut %s", t.Type.toString())
+		return fmt.Sprintf("&mut %s%s", lifetime, typeString), nil
```

Comment:
Should this be: `"&%s mut %s"`

https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=239b37af21ffdedb93be8e2a3d0c5da6